standards.md

```diff
@@ -248,13 +248,17 @@ PRs must adhere to the project's API stability policy.
 * Prefer small, well-factored packages with unit tests
 * Use meaningful package names (avoid util, helper, lib). See https://go.dev/blog/package-names
 * Only export functions that provide a meaningful API for consumers of the package
+  * Prefer unexported functions/fields/types, or using an [`internal` package](https://go.dev/doc/go1.4#internalpackages).
```

Comment:
it might be confusing to suggest using unexported functions, but also to suggest moving functions into a different package where they'll have to be exported. Internal packages are probably just better here?

Reply:
I wouldn't want to recommend an internal package if the code is only intended for use in one place and tightly coupled. My expectation is most things will probably go through a process like this, depending on what the intended audience is:

1. Start unexported.
2. Another package needs it, move to an internal package and export it.
3. Downstream users need it, move to an externally available package.